modules/income_limits/app/controllers/income_limits/v1/income_limits_controller.rb

```diff
@@ -32,9 +32,7 @@ def validate_zip_code
         zip = params[:zip].to_s
         zipcode_data = find_zipcode_data(zip)
         response = false
-        if zipcode_data.present?
-          reponse = true
-        end
+        response = true if zipcode_data&.county_number
```

Comment:
**Suggestion:** the response assignment could be refactored to use a ternary operator.

```ruby
response = zipcode_data&.county_number ? true : false
```